bundles/org.openhab.voice.mimictts/src/main/resources/OH-INF/i18n/mimictts_fr.properties

```diff
@@ -0,0 +1,13 @@
+voice.config.mimictts.audioVolatility.label = Volatilité Audio
+voice.config.mimictts.audioVolatility.description = Quantité de bruit ajouté à l'audio généré (0-1). Peut aider à masquer les artefacts du modèle de voix. Les modèles à plusieurs élocuteurs sonnent mieux avec une quantité réduite de bruit, contrairement aux modèles à élocuteur unique.
+voice.config.mimictts.group.tts.label = Configuration TTS
+voice.config.mimictts.group.tts.description = Paramètres pour l'API du TTS Mimic 3. 
+voice.config.mimictts.phonemeVolatility.label = Volatilité Phonème
+voice.config.mimictts.phonemeVolatility.description = La quantité de bruit utilisé pour générer les durées des phonèmes (0-1). Permet une cadence d'élocution variable, avec une valeur proche de 1 signifiant plus de variabilité. Les modèles à plusieurs élocuteurs sonnent mieux avec une quantité réduite de bruit, contrairement aux modèles à élocuteur unique. 
+voice.config.mimictts.speakingRate.label = Vitesse d'élocution
+voice.config.mimictts.speakingRate.description = Controle la vitesse d'élocution. Une valeur de 1 correspond à la vitesse d'entrainement du jeu de données. Moins de 1 est plus rapide, and plus de 1 est plus lent.
+voice.config.mimictts.url.label = URL
+voice.config.mimictts.url.description = L'URL pour joindre l'API de Mimic.
+
+# service
+service.voice.mimictts.label = Mimic Text-to-Speech
```

Comment:
Note that the translations are expected to be done in Crowdin. But ok, I will upload these translations in Crowdin once the PR is merged.
For consistency, please translate rather like that: Synthèse vocale Mimic

Reply:
Oh OK, I wasn't aware of that. I thought there was some kind of sync with Crowdin.